staging/src/k8s.io/apiserver/pkg/server/filters/waitgroup.go

```diff
@@ -24,20 +24,29 @@ import (
 	"k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
-	utilwaitgroup "k8s.io/apimachinery/pkg/util/waitgroup"
 	"k8s.io/apiserver/pkg/endpoints/handlers/responsewriters"
 	apirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+type RequestWaitGroup interface {
```

Comment:
Let's move this refactoring to a separate PR and have that merged immediately (please assign it to me).

Reply:
yes, https://github.com/kubernetes/kubernetes/pull/114970 has merged